### ISSUE TITLE
Bearer middleware with Cloudflare Workers environment variables only works when multiple middlewares are defined

### ISSUE THREAD

**Issue by joaocstro** [honojs/hono#509]
I'm trying to use the bearer auth middleware in a worker by providing the token from an environment variable. Currently i got the following code to work, but it only seems to work when i have multiple middlewares defined, otherwise it appears to be ignored.

```typescript
app.use('*',
    async (context, next) => {
        const auth = bearerAuth({ token: context.env.ARTICLE_MANAGER_TOKEN });
        await auth(context, next);
    },
    async (context, next) => {
        console.log('test');
        await next()
    }
);
```

**Comment by yusukebe** [comment]
Hi @joaocstro 

Thank you for creating the issue. This seems to be a bug. I'll investigate it. Please wait!

**Comment by yusukebe** [comment]
@joaocstro 

This issue may be fixed by #513 

The next version will include this patch. Thank you!

**Comment by joaocstro** [comment]
@yusukebe Nice! Thanks for the quick support!
